docs/using-gitbase/functions.md

```diff
@@ -19,6 +19,7 @@ To make some common tasks easier for the user, there are some functions to inter
 |`uast_extract(blob, key) text array`| extracts information identified by the given key from the uast nodes.                                       |
 |`uast_mode(mode, blob, lang) blob`| returns a node array of UAST nodes specifying its language and mode (semantic, annotated or native).          |
 |`uast_xpath(blob, xpath) blob`| performs an XPath query over the given UAST nodes.                                                                |
+|`uast_imports(blob) text array`| get all imports given the specified UAST blob. |
```

Comment:
nitpick: the rest of the table is ordered alphabetically, it would be better to move this below `uast_extract`.